docs/formatter/black.md

```diff
@@ -650,52 +714,3 @@ assert AAAAAAAAAAAAAAAAAAAAAA.bbbbbb.fooo(
     foooooo + 1
 ) * foooooo * len(list(foo(bar(4, foo), foo)))
 ```
-
-### The last context manager in a `with` statement may be collapsed onto a single line
```

Comment:
Black removed support for [pre Python 3.9](https://github.com/psf/black/pull/4452). We can't deviate from something that no longer exists ;)